samples/msal-browser-samples/VanillaJSTestApp2.0/app/facebook-sample/authConfig.js

```diff
@@ -0,0 +1,71 @@
+// Config object to be passed to Msal on creation
+const msalConfig = {
+    auth: {
+        clientId: "ENTER_CLIENT_ID_HERE",
+        authority: "https://www.facebook.com",
+        knownAuthorities: ["www.facebook.com"],
```

Comment:
nit:
```suggestion
        knownAuthorities: ["facebook.com"],
```

Reply:
This throws an "authority not recognized" error

Reply:
Try removing it from the authority field as well. It's not that important, if you want to leave it that's totally fine.